modules/basics/src/main/java/com/opengamma/strata/basics/schedule/StandardRollConventions.java

```diff
@@ -33,6 +33,11 @@ public LocalDate adjust(LocalDate date) {
       ArgChecker.notNull(date, "date");
       return date.withDayOfMonth(date.lengthOfMonth());
     }
+
+    @Override
+    public int getDayOfMonth() {
+      return 31;
```

Comment:
This seems slightly odd. I'm sure it works, but is it worth a comment explaining what's going on?